galette/templates/default/pages/contributions_list.html.twig

```diff
@@ -299,13 +312,17 @@
                 {% else %}
                     {{ ordre + 1 + (filters.current_page - 1) * numrows }}
                 {% endif %}
+
                 {% if (login.isAdmin() or login.isStaff()) and mode != 'ajax' %}
                     {% if contribution.isTransactionPart() %}
                         <a
                             href="{{ url_for("editTransaction", {"id": contribution.transaction.id}) }}"
                         >
-                            <i class="ui sitemap icon tooltip" aria-hidden="true"></i>
-                            <span class="ui special popup">{{ _T("Transaction: %s")|replace({"%s": contribution.transaction.description}) }}</span>
+                            <i class="ui sitemap {% if contribution.transaction.getMissingAmount() == 0 %}blue{% else %}orange{% endif %} icon tooltip" aria-hidden="true"></i>
+                            <span class="ui special popup">
+                                {{ _T("Edit related transaction") }}
```

Comment:
When possible, I'd prefer to use existing string:
```suggestion
                                {{ _T("Edit transaction #%id")|replace({"%id": contribution.transaction.id}) }}
```

Reply:
To be honest, I wonder if this setting is really useful. I see no real reason to hide database ID and replace it with a positional one. This has existed for years; but I was recently (this afternoon :D) wondering if I'll not remove it. That would make things more simple.

I try to add as few translations as possible; existing strings may be translated in some languages already; new strings probably won't be for 1.2.0 release.

Reply:
> > I was recently (this afternoon :D) wondering if I'll not remove it. That would make things more simple.
> 
> I agree 😆

https://bugs.galette.eu/issues/1949 :)

> 
> > I try to add as few translations as possible; existing strings may be translated in some languages already; new strings probably won't be for 1.2.0 release.
> 
> Let's keep the existing one then !

:+1: